pkg/chartutil/capabilities.go

```diff
@@ -32,22 +35,6 @@ const (
 	k8sVersionMinor = 20
 )
 
-var (
-	// DefaultVersionSet is the default version set, which includes only Core V1 ("v1").
-	DefaultVersionSet = allKnownVersions()
```

Comment:
so unfortunately we need to keep these in for the time being. As per our contributing policy, we try not to make any breaking changes to the API until the next major release. Many users may not be as concerned about race conditions, but they would be concerned about their apps breaking in a 3.x release!

I would suggest adding these back, but then adding a comment above informing the reader that this has been deprecated in favour of `DefaultVersionSet()`. Same for `DefaultCapabilities`.